draft-ietf-quic-tls.md

```diff
@@ -2299,17 +2307,23 @@ o:
 : The amount of offline ideal cipher queries made by an adversary.
 
 The analyses that follow rely on a count of the number of block operations
-involved in producing each message. For simplicity, and to match the analysis of
-other AEAD functions in {{AEBounds}}, this analysis assumes a packet length of
-2^10 blocks; that is, a packet size limit of 2^14 bytes.
+involved in producing each message. This analysis is performed for packets of
+up to 2^11 (l = 2^7) and 2^16 (l = 2^12). A size of 2^11 is expected to be a
+limit that matches common deployment patterns, whereas the 2^16 is the maximum
+possible size of a QUIC packet. Only endpoints that strictly limit packet size
+can use the larger limits that are derived using the smaller packet size.
```

Comment:
limit is being used for multiple things here -- these limits and also packet size limit. 
```suggestion
can use the larger confidentiality and integrity limits that are derived using the smaller packet size.
```